modules/terraform/destroy.go

```diff
@@ -13,7 +14,25 @@ func Destroy(t *testing.T, options *Options) string {
 	return out
 }
 
+// DestroyAll runs terragrunt destroy with the given options and return stdout.
+func DestroyAll(t *testing.T, options *Options) string {
```

Comment:
Can you call this `TgDestroyAll` for consistency with the `ApplyAll` naming pattern?